metpy/calc/basic.py

```diff
@@ -62,12 +62,16 @@ def wind_direction(u, v):
         Wind component in the X (East-West) direction
     v : array_like
         Wind component in the Y (North-South) direction
+    return_calm_as_zero: bool, optional
```

Comment:
I'm not sure I like adding a flag for this. I'd be ok with just moving to `0` for calm. @jrleeman ?

Reply:
If it doesn't break plotting things, I'd rather have calm return NaN... a direction with no wind is something only a METAR could love.